mappings/net/minecraft/client/gui/screen/ingame/CraftingScreen.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_793 net/minecraft/client/gui/screen/ingame/CraftingTableScreen
+CLASS net/minecraft/class_793 net/minecraft/client/gui/screen/ingame/CraftingScreen
```

Comment:
bad. less yarn is not good. revert this

Reply:
![image](https://user-images.githubusercontent.com/62552372/96240596-b92eff80-0fbe-11eb-933f-e49135582fa2.png)